wfsim/core/s2.py

```diff
@@ -221,7 +233,7 @@ def get_electron_yield(n_electron, xy_int, z_int, config, resource):
         # Absorb electrons during the drift
         electron_lifetime_correction = np.exp(- 1 * drift_time_mean /
                                               config['electron_lifetime_liquid'])
-        cy = config['electron_extraction_yield'] * electron_lifetime_correction
+        cy = config['g2_mean']*resource.s2_correction_map(xy_int)*electron_lifetime_correction/resource.se_gain_map(xy_int)
```

Comment:
I could not trace back to this `'g2_mean'` is it supposed to be in fax config on your branch?

Reply:
Indeed it is. I just changed my fax config branch to match this. Thanks for pointing it out, I was simply using the fax override to do this before